zerver/lib/test_helpers.py

```diff
@@ -329,14 +361,17 @@ def assert_length(self, queries, count, exact=False):
                                "len(%s) == %s, > %s" % (queries, actual_count, count))
 
     def assert_json_error_contains(self, result, msg_substring, status_code=400):
+        # type: (HttpResponse, str, int) -> None
         self.assertIn(msg_substring, self.get_json_error(result, status_code=status_code))
 
     def fixture_data(self, type, action, file_type='json'):
+        # type: (str, str, str) -> str
         return open(os.path.join(os.path.dirname(__file__),
                                  "../fixtures/%s/%s_%s.%s" % (type, type, action, file_type))).read()
 
     # Subscribe to a stream directly
     def subscribe_to_stream(self, email, stream_name, realm=None):
+        # type: (str, str, Any) -> None
```

Comment:
the third argument is I think an Optional[Realm].  Also this code is clearly broken if you pass realm, since it ignored that argument; I think it's intended to have an `if realm is None` check around the first line...